global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "7.0.400",
+    "version": "7.0.0",
     "rollForward": "latestFeature",
     "allowPrerelease": false
```

Comment:
Please note that 7.0.100 is the first (stable) 7.x SDK version and `allowPrerelease` is set to the default value, so can be omitted:
- https://dotnet.microsoft.com/en-us/download/dotnet/7.0
- https://learn.microsoft.com/en-us/dotnet/core/tools/global-json